libs/libmbim/Makefile

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libmbim
+PKG_VERSION:=1.18.2
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://anongit.freedesktop.org/libmbim/libmbim
```

Comment:
Why not just use a normal tarball? Would simplify the Makefile.

Reply:
It gets one from freedesktop.org.  In this way we can track the project more easily at the cost of a slightly more complicated Makefile.
```
PKG_SOURCE_PROTO:=git
PKG_SOURCE_URL:=git://anongit.freedesktop.org/libmbim/libmbim
PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
PKG_SOURCE_VERSION:=2028fd2f2effbcf80d0503cc163ac8f6316d8359
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
```

Reply:
If we want to have stable updates only included in the openwrt package repostory, we could always point to the actual release tarballs, e.g. in https://www.freedesktop.org/software/ModemManager/ModemManager-1.10.0.tar.xz

My original packaging used git commit ids because I sometimes use it to flag unreleased versions.

Reply:
The original tarballs are preferred. Upstream has frequent updates.